src/execution.ts

```diff
@@ -764,7 +767,13 @@ function compileObjectType(
       }" but got: $\{${GLOBAL_INSPECT_NAME}(${originPaths.join(".")})}.\``
     )}), null) :`;
   }
-  body += "{";
+
+  /**
+   * iife is used to optionally attach different fields
+   * based on skip/include directives.
+   */
+  const iifeBody = genFn();
```

Comment:
I think we should go for a different approach where we set it to undefined and avoid iife. The benchmarks show a regression even though they do not use skip/include.

Reply:
done 👍